modules/dmrpp_module/.gitignore

```diff
@@ -9,3 +9,4 @@
 /ngap_build_dmrpp
 /bes_default_conf.h
 /foo.dmrpp
+/airs.dmrpp
```

Comment:
I think we shouldn't ignore these generated .dmrpp files but rather we might remove them for the repo

Reply:
alright, i'll make those changes this morning